### PR TITLE
Adding readme and contribution guidelines

### PR DESCRIPTION
**Pull request by impulse**

### CODE REVIEW

**Comment by luanpotter** [comment]
This looks good, @impulse ! I have one suggestion: for me flame-awesome would be akin to a flutter-awesome or anything else, containing several categories: one for sure is games made with flame (that would have), but what about adding stuff like tutorials/articles/videos/courses and libraries (like flame_gamepad and other plugins that are used alongside flame)?

**Comment by impulse** [comment]
> This looks good, @impulse ! I have one suggestion: for me flame-awesome would be akin to a flutter-awesome or anything else, containing several categories: one for sure is games made with flame (that would have), but what about adding stuff like tutorials/articles/videos/courses and libraries (like flame_gamepad and other plugins that are used alongside flame)?

I was thinking about adding a Library / Framework section. The tutorials part however would overlap with whats available on the main repo though. Maybe consider moving this here all together @luanpotter ?

**Comment by luanpotter** [comment]
@impulse right now we only have @japalekhin series on the main repo, I believe, and there are more we could add here (hopefully many many more in the future ahha). Then, on the main repo, we can for sure link back here and either have or not a link to a more hand-picked beginner's friendly starter guide like Alekhin's.

**Comment by impulse** [comment]
@luanpotter it really depends on how much you want to bloat the main repos `README`.

My preference would be to only maintain it in one place -> move it to this repo. 
We can also keep the series for now until there are way more tutorials on flame-awesome.

Could look like the following: 
## Getting started

Check out [flame-awesome](https://github.com/flame-engine/flame-awesome) for a curated list of great tutorials and projects.

File: contributing.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+`- [**My Plugin/Library**](repository url) - Short description by [Name](url)`
+
+## Games
+
+1. Open Source is a great learning resource, thus publicly available projects will be listed first.
+1. Make sure to add your Game to the correct Genre.
+1. If you add a new Genre, please order them A-Z.
+
+#### Open Source
```
Comment (japalekhin) [inline contributing.md]: not that big of a deal but why jump from `h2` to `h4`?

Comment (impulse) [reply contributing.md]: Oh didn't notice that, fixed ty. 13b3b88

File: contributing.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
+# Contributing Guide
+
+## General
+
+1. If you think something belongs in the wrong category, or think there needs to be a new category, feel free to edit things.
+1. Always add your items to the end of a list. The order is first-come-first-serve.
+1. Always add the "Add your item" to the end of each list
```
Comment (luanpotter) [inline contributing.md]: 1, 2, 3?

File: contributing.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+`- [Add Your Item](https://github.com/flame-engine/flame-awesome/pulls)`
+
+## Articles
+
+You should put your project in this category if the following conditions are true:
```
Comment (luanpotter) [inline contributing.md]: your article/video/tutorial

File: readme.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+- [Add Your Item](https://github.com/flame-engine/flame-awesome/pulls)
+
+## Contributing
+
+[Please read the contribution guidelines](https://github.com/flame-engine/flame-awesome/blob/master/contributing.md)
+
+## License
+
+MIT
```
Comment (luanpotter) [inline readme.md]: link to the LICENSE file

File: contributing.md (hunk 2 above)
Comment (japalekhin) [reply contributing.md]: MD allows (i think recommends) this. in fact, i'd use 0, 0, 0 if not for the fact that other MD parsers use the first number as the first number in the sequence. 10, 0, 0 would result in 10, 11, 12

File: contributing.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
+# Contributing Guide
+
+## General
+
+1. If you think something belongs in the wrong category, or think there needs to be a new category, feel free to edit things.
+1. Always add your items to the end of a list. The order is first-come-first-serve.
+1. Always add the "Add your item" to the end of each list
+
```
Comment (erickzanardo) [inline contributing.md]: Don't forget to update links to match the new repository name

File: contributing.md (hunk 2 above)
Comment (luanpotter) [reply contributing.md]: really? I did not know that! thanks for clearing it up! guess it's easier to add/remove items this way

File: contributing.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+### Open Source
+
+You should put your project in this category if the following conditions are true:
+
+- Made with the Flame Engine
+- The project contains a `LICENSE` file
+- The documentation `README` contains a description of the project, illustration of the project with a demo or screenshots.
+- The documentation `README` is in English.
+- The project accepts contributions.
```
Comment (erickzanardo) [inline contributing.md]: I am not sure that this should be a requirement, I mean the author could not want to accept contributions, but would like to keep open source so people could learn from it?

**Comment by impulse** [comment]
@luanpotter @erickzanardo we need a different license though.

From awesome list guidelines:
```
Has an appropriate license.
That means something like CC0, not a code licence like MIT, BSD, Apache, etc.
WTFPL and Unlicense are not acceptable licenses.
If you use a license badge, it should be SVG, not PNG.
To verify that you've read all the guidelines, please add comment with just the word unicorn.
```

https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md

**Comment by impulse** [comment]
Changed it, let me know if thats fine so we can merge 👍 .

**Review by dariel97 — APPROVED** [review]

**Pull request merged**